src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/QRScanner/ExposureSubmissionQRScannerViewController.swift

```diff
@@ -89,7 +89,7 @@ final class ExposureSubmissionQRScannerViewController: UIViewController {
 		let flashButton = UIButton(type: .custom)
 		flashButton.imageView?.contentMode = .center
 		flashButton.addTarget(self, action: #selector(didToggleFlash), for: .touchUpInside)
-		flashButton.setImage(UIImage(named: "bolt"), for: .normal)
+		flashButton.setImage(UIImage(named: "flash_disabled"), for: .normal)
```

Comment:
it the images changed inside exposure submission flow as well?

Reply:
yes the designer told me the design is wrong in the exposure submission as well so i changed it